src/screens/SendFunds/04-Summary.js

```diff
@@ -140,6 +148,16 @@ export default function SendSummary({ navigation, route }: Props) {
           transaction={transaction}
           navigation={navigation}
         />
+        {gasPrice && gasPrice instanceof NotEnoughGas ? (
```

Comment:
mmh IMO we shouldn't do this specific code.

we should use

```
error = status.errors[Object.keys(status.errors)[0]]
```

because any error is a case of blocker and we should show it to user (show the first one at least)

Reply:
This is a new block specifically made for this error, I have not touch the rest of the UI

Reply:
mmh to be confirmed but imo we want to do such thing whatever the error is. for instance, we have same issue on XRP if you go increase the fees from that screen. which would not be caught here.

Reply:
I still don't know what do with this then ? @gre @cthepot-ledger @khalilbenihoud

Reply:
even if this could be enough as-is, I still think it would be better to generalize this to the equivalent of the logic of the NEXT that I think with this idea of taking first error in status.errors

this would address it more generically like for XRP and to me this is compatible with the idea of any error as here you are already using TranslatedError

Reply:
to summarize:
in the Summary step, there is rarely a case where an error exist, like 99% of the time, status.errors is settled to {} (no errors). but when it happens, we want that error to be displayed.

Reply:
👍